src/app/pages/OriginsLaunchpad/pages/SalesDay/pages/BuyStep/components/BuyDialog/components/TxDialog/index.tsx

```diff
@@ -23,44 +23,41 @@ interface ITxDialogProps {
 }
 
 export const TxDialog: React.FC<ITxDialogProps> = (props: ITxDialogProps) => {
```

Comment:
```suggestion
export const TxDialog: React.FC<ITxDialogProps> = (props) => {
```

Reply:
You don't need to specify type of props here since you specify it in the generics.